File: my-app (2)/my-app/src/main/java/io/bootify/my_app/componentes/PagAdmin.js
import React from 'react';

function PagAdmin() {
    const handleCreateUser = async () => {
        const newUser = {
            nombre: 'nuevoUsuario',
            contraseña: 'password123',
            esAdmin: false
        };
        await fetch('/api/users/create', {
            method: 'POST',
            headers: {
                'Content-Type': 'application/json',
            },
            body: JSON.stringify(newUser),
        });
    };

    const handleDeleteUser = async (id) => {
        await fetch(`/api/users/delete/${id}`, {
            method: 'DELETE',
        });
    };

    return (
        <div>
            <h1>Admin Screen</h1>
            <button onClick={handleCreateUser}>Crear Usuario</button>
            <button onClick={() => handleDeleteUser(1)}>Borrar Usuario</button>
        </div>
    );
}

export default PagAdmin;

File: my-app (2)/my-app/src/main/java/io/bootify/my_app/componentes/PagAdmin.java
package io.bootify.my_app.componentes;

import React from 'react';

function PagAdmin() {
    const handleCreateUser = () => {
        // Lógica para crear usuario
    };

    const handleDeleteUser = () => {
        // Lógica para borrar usuario
    };

    return (
            <div>
            <h1>Admin Screen</h1>
            <button onClick={handleCreateUser}>Crear Usuario</button>
            <button onClick={handleDeleteUser}>Borrar Usuario</button>
        </div>
    );
}

export default AdminScreen;
